src/libraries/ExecutorParserLib.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+/**
+ * @title ExecutorParserLib
+ * @notice Library to parse additional data from Executor messages.
+ */
+library ExecutorParserLib {
+
+  /// @notice Parses the message ID from `msg.data`.
+  /// @return _messageId The bytes32 message ID uniquely identifying the message that was executed
+  function messageId() internal pure returns (bytes32 _messageId) {
```

Comment:
our convention is to name return vars when there is more than one, otherwise don't name them

Reply:
Fixed by: [a3f4729](https://github.com/GenerationSoftware/erc5164-interfaces/pull/1/commits/a3f47291ed7ca85af67ad91bcfbd4c433d1500a2)